src/pages/certifications/front-end-libraries/react/compose-react-components/index.md

```diff
@@ -1,10 +1,32 @@
 ---
-title: Compose React Components
+title:Compose React Components
 ---
-## Compose React Components
+##Compose React Components
```

Comment:
Hi @angelobarbano 

You need to add a space between `##` and the first letter of the heading. It's not rendering correctly as well.